kubeflow/gcp/basic-auth-ingress.libsonnet

```diff
@@ -430,6 +436,45 @@
     ),
     cloudEndpoint:: cloudEndpoint,
 
+    local istioMappingSvc = {
+      apiVersion: "v1",
```

Comment:
This service seems not matching to any deployment. Is it only for the annotation?

Reply:
yes.